enhancements/network/service-egress-traffic-steering.md

```diff
@@ -63,20 +69,24 @@ The ingress part will be handled by a LoadBalancer provider, such as MetalLB, th
 Taking MetalLB as an example for a LoadBalancer provider, the user will need to create their `L2Advertisement` and/or `BGPAdvertisements` with the `nodeSelector` field pointing to that label. That way only the node holding the label will be used for announcing the LoadBalancer service ingress IP.
 It is worth noting that in MetalLB's case, a given LoadBalancer service can be announced by multiple L2 and BGP advertisements, possibly being (even accidently) announced from multiple nodes. For our use-case the user MUST take care of configuring their MetalLB resources in a way that the service is announced only by the node holding the label - a full example is detailed in [Usage Example](#Usage-Example).
 
-To achieve these goals, we introduce a new resource `EgressService` for users to create alongside LoadBalancer services which can be either empty or contain optional fields:
-- `nodeSelector`: allows limiting the nodes that can be selected to handle the service's traffic.
+### Implementation Details/Notes/Constraints
+
+A new resource `EgressService` is supported alongside LoadBalancer services with the following optional fields:
+- `snat`: Determines if the source IP of egress traffic originating from the pods backing the Service
+should be set to its ingress IP.
+When the field is not specified the source IP is set according to the interface of the Network, leveraging the masquerade rules that are already in place.
+  - `nodeSelector`: Allows limiting the nodes that can be selected to handle the service's traffic.
```

Comment:
so this takes effect even when load-balancer-snat-ip is false and we dont necessarily need to "select a single egress node"?

Reply:
right, added `Network without SNAT` section